docs/ehrql-new-tutorial-1a.md

```diff
@@ -59,110 +39,35 @@ Here, we import two components of Data Builder:
-    it is the call that appears closest to the end of the dataset definition
-    that takes effect.
-
-    In future, calling `set_population()` more than once will cause a dataset definition to fail;
-    see the associated [Data Builder](https://github.com/opensafely-core/databuilder/issues/775) issue.
-
-!!! todo
-
-    Would it be simpler here to say:
-    "you probably should only have one dataset".
-    Are there cases where you might need multiple datasets within a definition?
-
-!!! todo
-
-    Consider using code annotations to describe the code.
-
-#### What happens when Data Builder generates a dataset?
-
-When the dataset definition is used by Data Builder to generate a dataset:
-
-1. The dataset definition is validated
-   to ensure that the resulting database query would be valid.
-2. A database query suitable for the specified database is created.
-3. The query is submitted to the database.
-4. Provided the query is successful, the query creates an output.
-
-In writing a *dataset definition* then,
-what we are really writing is a *database query*.
-Data Builder transforms the dataset definition into the appropriate database query,
-for the specific database.
-
-!!! note
-
-    You can see the database query that a dataset definition will generate
-    via the `databuilder --dump-dataset-sql` command.
-
-    For the minimal example `1a_minimal_dataset_definition.py` above,
-    the underlying SQL query generated is:
-
-    ```sql
-    SELECT patients.patient_id AS patient_id
-    FROM patients
-    WHERE CAST(STRFTIME('%Y', patients.date_of_birth) AS INTEGER) >= 2000;
-    ```
+## Your turn
+Run the dataset definition by:
 
-    !!! todo
+```
+opensafely exec databuilder:v0 generate-dataset "1a_minimal_dataset_definition.py" --dummy-tables "example-data/minimal/" --output "outputs.csv"
+```
 
-        This SQL should be kept updated.
+or if you are using `project.yaml`:
 
-That database might be:
+```
+opensafely run extract_1a_minimal_population
+```
 
-* a local database used for testing,
-  such as in this tutorial
-* an OpenSAFELY backend,
-  when submitting jobs to OpenSAFELY
+!!! question
 
-There are two important implications of how this Data Builder's process works:
+    Can you modify the dataset definition so that the output shows:
 
-1. **Queries do not have to be written in a specific query language tailored to a specific database.**
-   The compatibility of a dataset definition depends only
-   on whether all the data tables used are available in the database being queried.
-2. **All the data processing requested happens at the database
-   after the dataset definition is processed in its entirety.**
-   This is different from a more typical interactive data analysis in Python, Stata or R
-   where you load some data,
-   then perform computations on that data as each line of the analysis code runs.
+    1. Patients that were born before 1980?
+    2. Patient that were born between 1980 and 2000?
```

Comment:
thought: yes, I thought it was too simple an example to even prompt for suggestions, but I like this :+1: